Sources/SnapshottingTests/AccessibilityPreviewTest.swift

```diff
@@ -9,42 +9,71 @@ import Foundation
 import MachO
 import XCTest
 
-// This is an XCUITest that uses XCUIApplication.performAccessibilityAudit to test previews
+/// A base class for conducting accessibility audits on SwiftUI previews using XCTest and XCUITest.
+///
+/// This class provides a framework for running accessibility audits on SwiftUI previews.
```

Comment:
Maybe an explanation of how to use it explaining it needs to be subclasses would be helpful, since above you call it a "base class"